packages/l10n/data/time-zones/de.json

```diff
@@ -4,8 +4,8 @@
     "abbr": "-12",
     "offset": "-12:00"
   },
-  "Pacific/Niue": {
-    "name": "Pacific/Niue",
+  "Etc/GMT+11": {
```

Comment:
I'm not sure why this has changed.
But my deps are updated, and this was generated when I ran `generate-data`.
Will be very helpful if someone regenerated locally to make sure that this is how it is supposed to be.

Reply:
I regenerated the data. I think you might need to re-install all deps.

Reply:
Thanks a lot.
Will keep that in mind next time.